server/src/main/java/edu/jhu/server/Player.java

```diff
@@ -71,6 +72,9 @@ public void onWebSocketError(Throwable cause) {
   @Override
   public void onWebSocketText(final String message) {
     log("Message: " + message);
-    game.handleEvent(new JSONObject(message));
+    JSONObject JSONMessage = new JSONObject(message);
+    JSONMessage.append("author", this.tag);
+    JSONMessage.put("author", this.tag);
```

Comment:
can you move this author stuff to Game::handleEvent? all event specific stuff should go in Game